tests/unittests/core/worker/test_consumer.py

```diff
@@ -59,7 +59,7 @@ def mock_popen(self, *args, **kwargs):
     shutil.rmtree(trial.working_dir)
 
 
-def setup_code_change_mock(config, monkeypatch, ignore_code_changes):
+def setup_code_change_mock(storage, config, monkeypatch, ignore_code_changes):
```

Comment:
```suggestion
def setup_code_change_mock(storage: dict, config: dict, monkeypatch, ignore_code_changes: bool):
```

Reply:
Storage is not a dict here. If you don't mind I'd skip on this one to wrap up this PR.